walk-in-interview/src/main/webapp/account/firebase-auth.js

```diff
@@ -92,33 +153,17 @@ Auth.addPhoneAuthUI = (elementId, successPath) => {
       signInSuccessWithAuthResult: (authResult, redirectUrl) => {
         // TODO(issue/89): add new user pages for name/skills
         // for now, this will only redirect to homepage for exisiting users
-        return !authResult.additionalUserInfo.isNewUser;
+        if (authResult.additionalUserInfo.isNewUser) {
+          alert(newUserInfo);
```

Comment:
do you still need this? (seems that it was for debugging purpose)

Reply:
this is to inform the user, not for debugging purpose